src/v/raft/consensus.h

```diff
@@ -252,21 +253,22 @@ class consensus {
         return _configuration_manager.wait_for_change(last_seen, as);
     }
 
-    ss::future<> step_down(model::term_id term) {
-        return _op_lock.with([this, term] {
+    ss::future<> step_down(model::term_id term, std::string_view ctx) {
+        return _op_lock.with([this, term, ctx] {
```

Comment:
i'd be tempted to capture a copy instead of the view. sometime in the future if someone ever does something like `step_down(term, fmt::format("something {}", blah))` then this becomes a UAF issue.